assets/js/modules/adsense/components/settings/AdBlockingRecoverySetupCTANotice.test.js

```diff
@@ -48,8 +51,17 @@ mockTrackEvent.mockImplementation( () => Promise.resolve() );
 describe( 'AdBlockingRecoverySetupCTANotice', () => {
 	mockLocation();
 
+	let registry;
 	beforeEach( () => {
 		mockTrackEvent.mockClear();
+		mockSurveyEndpoints();
+		registry = createTestRegistry();
```

Comment:
The changes to this file are best viewed ignoring whitespace, but aside from the added test are mostly refactoring. Also, utilities like `provideModules` weren't necessary and there was a fair bit of duplication due to not setting up the registry here in the `beforeEach`.